ideascube/utils.py

```diff
@@ -95,5 +95,6 @@ def sanitize_tag_name(tag_name):
     return tag_name
 
 def tag_splitter(tag_string):
-    tags = set(t.strip() for t in re.split(r'[;,]+', tag_string) if t.strip())
+    tags = set(sanitize_tag_name(t) for t in re.split(r'[;,]+', tag_string))
+    tags = filter(None, tags)
```

Comment:
Aren't you the one who convinced me to stop using `map`, `filter` and their friends? 😛